configs/atlassian-backup.conf

```diff
@@ -14,6 +17,18 @@ JIRA_BACKUP_DIR='/atlassian/backups/jira'
 JIRA_ATTACHMENTS_PATH='/local/jira/jira-6.4.1/data'
 JIRA_CONFIGURATION_PATH=''
 
+# JIRA DB Connection Information
+# The following vars are used to connect to your JIRA DB instance. 
+# The mysql information is only used for MySQL connections, however JIRA_DB_NAME is used for both PG and MySQL.
+# Chose the backend for your database, should be either 'pg' for PostgreSQL or 'mysql' for MySQL.
+JIRA_DB_BACKEND='pg'
+# JIRA_DB_BACKEND='mysql'
+# A database user that has access to your JIRA MySQL DB
+JIRA_MYSQL_USER=''
+# The password for the user that has access to your JIRA MySQL DB
+JIRA_MYSQL_PASS=''
+# The MySQL host that hosts your JIRA DB
+JIRA_MYSQL_HOST=''
 # JIRA Database
 JIRA_DB_NAME='jiradb_6_4'
```

Comment:
I recommend (but I do not demand) removing "_6_4".  Database names in Atlassian documentation do not include the version, but I appended it anyway.

 As it turns out, it is unnatural to change the database name when upgrading, so Atlassian is right.

Reply:
I've removed the version numbers from the db names. I also updated the Jira and Confluence home paths to more recent defaults.